lib/glimesh/chat.ex

```diff
@@ -193,6 +193,9 @@ defmodule Glimesh.Chat do
     end
   end
 
+  def user_in_message(chat_message, user) do
```

Comment:
```suggestion
  def user_in_message(username, chat_message) do
```
Might add some clarity to this function to flip the ordering to match the function name, and expand user to username.

Reply:
fixed